communication/blog.md

```diff
@@ -0,0 +1,43 @@
+# Blog posts
+
+Our most common form of communication is via [the blog at 2i2c.org/blog](https://2i2c.org/blog).
+It is managed by our [Hugo website repository](https://github.com/2i2c-org/2i2c-org.github.io).
+
+
+## Checklist
+
+Here's a blog-post checklist to help us keep track of the major actions needed to publish a new post.
+Open an issue about your post and copy/paste this checklist into it.
+
+```
+- [ ] Write draft of blog post in markdown
+- [ ] Open a PR to the 2i2c.github.io repository and ask for feedback
+- [ ] Publish the post
+- [ ] Send out a series of tweets from the `2i2c_org` account that summarizes the post.
+- [ ] Tell the team about it and re-tweet!
+- [ ] Decide if this post is worthy for a mailing list update
+- [ ] If so, go to mailchimp.com and create a new email campain
+- [ ] Copy the post's content, paste it into the mailchimp interface
+- [ ] Send off the mailchimp campain
+- [ ] You're done! 🎉
+```
+
+## What to post on the blog
+
+Generally speaking, we can post as often as we wish.
+Our blog should be a running heartbeat of our organization, and it is OK if the information on there isn't perfect.
+Be creative and experiment with formats that you think might be interesting or fun.
+If there's something particularly important, we can signal-boost it with our [Mailing list](mailinglist.md).
+
+Here are a few ideas for blog post topics for inspiration:
+
+* **2i2c organizational updates**. Provide updates about important organizational decisions and changes. For example:
+    * Major changes to strategy
+    * Governance or other organizational changes
+    * New hires
+* **Major project updates**. For major projects and collaborations, we should provide regular updates about the work that we've done. This shouldn't frame 2i2c as the sole producer for these projects (unless we really are the sole producer), but demonstrate how 2i2c has contributed to the overall effort. For example:
+    * Major deliverables that we’ve met on a project
+    * New collaborations
+* **Open source updates**. If we do particular service work, or make a strong technical improvement in an open source project, we should tell the world about it. 
+* **Updates from the hubs pilot**. As our pilot hub infrastructure grows and evolves, and as we serve more communities with it, we should tell others about the impact we are having.
```

Comment:
Maybe we should not keep using the pilot hub concept? I think we have removed it from other places already...